opengrok-indexer/src/main/java/org/opengrok/indexer/history/HistoryEntry.java

```diff
@@ -178,6 +178,13 @@ public void setFiles(SortedSet<String> files) {
         this.files = files;
     }
 
+    /**
+     * The method is kept only for backward compatibility to avoid warnings when deserializing objects
+     * from the previous format. The tags were moved to the {@link History} class.
+     */
+    public void setTags(String tags) {
```

Comment:
Maybe use https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Deprecated.html with `since` and explicitly mention that it should be removed in the next (1.8.*) minor version?

Reply:
I suspected there is some better way, thanks.